### PR TITLE
adds the term hamburger-menu to the glossary

### PR DESCRIPTION
**Pull request by luanafaustini**
It has two files: content.lr (with the explanation of hamburger menu and a reference link) and an image (showing what is a hamburger menu using Tor Project website as example).

### CODE REVIEW

**Review by emmapeel2 — CHANGES_REQUESTED** [review]
The text is cool, I think the image will be better if its about the hamburguer menu of Tor Browser, as it is the one we talk about on the docs usually.

**Comment by luanafaustini** [comment]
could you show me which one you are talking about? I'll gladly change it, but I don't know if I understood.

**Comment by gusgustavo** [comment]
Hi @luanafaustini, you can see the menu here: https://tb-manual.torproject.org/static/images/security-settings-safest.png (on the top right corner)

**Comment by luanafaustini** [comment]
Ahh got it! thanks @gusgus 
I'm gonna update it right away.

File: content/glossary/hamburger-menu/content.lr [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+term: hamburger menu
+---
+definition:
+
+It's a icon with three horizontal lines usually on the top-left corner or top-right corner of the screen.
+By clicking or tapping on the icon, it reveals a menu with options or additional pages.
+
+Reference: https://en.wikipedia.org/wiki/Hamburger_button
+
+![hamburger-menu](hamburger-menu.png "Hamburger Menu")
```
Comment (gusgustavo) [inline content/glossary/hamburger-menu/content.lr]: Hey @luanafaustini, just a small modification:
The image should be moved to `assets/static/images/`
And you will need to modify this markdown path too.

**Comment by luanafaustini** [comment]
Hey @gusgus no problems! Gonna do that. I'm just not sure how the relative path goes. Should it be (assets/static/images/hamburger-menu.png "Hamburger Menu")? Thanks!

**Comment by gusgustavo** [comment]
Hi @luanafaustini, the relative path should look like [this example](https://raw.githubusercontent.com/torproject/support/master/content/onionservices/onionservices-5/contents.lr):
`Image of a green onion`

So, `Hamburger Menu` (and the file moved to /assets/static/images/)

**Comment by luanafaustini** [comment]
all done @gusgustavo

**Pull request merged**